### PR TITLE
trigger aws testing

### PR DESCRIPTION
**Pull request by Any2suited66**
## Description

Please provide:

- This integrates AWS Device farm into bitrise so our regression runs on real devices can be triggered automatically by bitrise
- Secrets were added to bitrise
- Tested and was able to successfully trigger a regression run using the `build-apps-android-external-e2e-full-regression-aws` pipeline 
- Cursor skill added to address co-pilot comments

## Testing

**Dev Testing (if applicable)**

- Provide steps to test the happy path of your feature
- Provide steps to test edge cases and error states
- Trigger a build on bitrise and reference it here
- Move the ticket into the "Testing" column on Jira

**QA Testing (if applicable)**

- There were lots of changes to the test code so it would work on real devices.

## Checklist

Please check all that apply (if applicable)

- [ ] I have performed a self-review of my code
- [ ] I have verified the code works
- [ ] I have included screenshots / videos of android and ios
- [ ] I have added testing steps
- [ ] I have added/updated necessary unit tests
- [ ] I have updated the documentation

### CODE REVIEW

File: packages/core-mobile/package.json [hunk 1]
```diff
@@ -261,6 +263,7 @@
   },
   "devDependencies": {
     "@avalabs/tsconfig-mobile": "workspace:*",
+    "@aws-sdk/client-device-farm": "3.986.0",
```
[the remark this answers is not in the document]
Comment (Any2suited66) [reply packages/core-mobile/package.json]: this is for security reasons we keep a static version

**Review by eunjisong — DISMISSED** [review]

**Review by eunjisong — DISMISSED** [review]

**Review by atn4z7 — APPROVED** [review]
excited to see this in action!!

**Review by eunjisong — APPROVED** [review]
